### PR TITLE
solved : https://issues.fluidproject.org/plugins/servlet/mobile#issue/FLUID-2797

### PR DESCRIPTION
**Pull request by SMe12435**
made changed to -

1.  **index.html** in /demos/pager/index.html 
2. **pager.css** in /demos/pager/css/pager.css

### CODE REVIEW

File: demos/pager/css/pager.css [hunk 1]
```diff
@@ -62,6 +62,7 @@
 }
 
 .demo-pager-bar  li  {
+
```
Comment (jobara) [inline demos/pager/css/pager.css]: Seems like some extra whitespace was added by mistake

**Comment by jobara** [comment]
@SMe12435 Thanks for submitting the PR. I don't think the absolute positioning works.

First of all, in the initial state the previous and next links do not look like they are associated with the page links, because they are so far away. And the alignment with the rest of the items on the page seems odd.
screen shot 2018-03-26 at 2 02 28 pm

Next, when the page width is decreased, the next link will be hidden offscreen, and the "items per page" drop down overlaps with the page links.
screen shot 2018-03-26 at 2 02 44 pm

**Comment by SMe12435** [comment]
Will make the required changes and get back to you soon.

**Comment by amb26** [comment]
Please let us know if you are still at work on the issue, otherwise we should close it soon - thanks.

**Pull request closed without merging**